cli/azd/internal/scaffold/spec.go

```diff
@@ -98,27 +98,56 @@ type ServiceSpec struct {
 	Name string
 	Port int
 
-	Env map[string]string
+	EnvironmentVariableInformation EnvironmentVariableInformation // todo: merge EnvironmentVariableInformation and env
 
 	// Front-end properties.
 	Frontend *Frontend
 
 	// Back-end properties
 	Backend *Backend
```

Comment:
Actually, the Frontend and Backend can be deleted, too. To avoid much conflict with the upstream, let's delete it after merged into upstream.